src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -108,6 +108,8 @@ export interface ICloseAllEditorsOptions {
 export interface IEditorReplacement {
 	editor: IEditorInput;
 	replacement: IEditorInput;
+	// Skips asking the user for confirmation and doesn't save the document. Only use this if you really need to!
```

Comment:
Can you do a JSDoc comment here instead so that people get this into the hover? JSDoc is `/** */`